service/history/workflow/context_util.go

```diff
@@ -119,6 +126,15 @@ func ConflictResolveWorkflowExecution(
 	}); err != nil {
 		return 0, 0, 0, err
 	}
+
+	engine := shard.GetEngine()
+	NotifyWorkflowSnapshotTasks(engine, resetWorkflowSnapshot)
+	NotifyWorkflowSnapshotTasks(engine, newWorkflowSnapshot)
+	NotifyWorkflowMutationTasks(engine, currentWorkflowMutation)
+	_ = NotifyNewHistorySnapshotEvent(engine, resetWorkflowSnapshot)
+	_ = NotifyNewHistorySnapshotEvent(engine, newWorkflowSnapshot)
+	_ = NotifyNewHistoryMutationEvent(engine, currentWorkflowMutation)
```

Comment:
why do _ = ?

Reply:
this function will notify waiting caller about workflow changes, e.g. long poll workflow events
if this fails (which is not expected) this should not block business logic

in the next PR i will probably add some logging here just in case